.template_version

```diff
@@ -1,3 +1,3 @@
 ---
 :remote: https://github.com/alphagov/tech-docs-template.git
-:revision: 22f782e1519eba5d324698059e54b10a1168569e
+:revision: head
```

Comment:
Do we know if it matters that this doesn't point to a specific commit?

Reply:
I'm not sure it does. I followed the instructions in the tech-docs-template repo for updating this, and this is the changes it made.

This isn't using submodules or anything like this, so I don't know what this is actually used for. When I updated the template, it pulled down all the files from the upstream repo, and wrote them into this repo prompting for any conflicts etc.

Reply:
I tried to find the relevant code in Middleman but Github search came up empty. Running the upgrade command on top of `head` worked fine so let's not worry about this.